src/models/auditModelGenerator.js

```diff
@@ -0,0 +1,149 @@
+// Load in our dependencies
+import { Model, DataTypes } from 'sequelize'; // eslint-disable-line import/no-import-module-exports
+import httpContext from 'express-http-context'; // eslint-disable-line import/no-import-module-exports
+import { auditLogger } from '../logger'; // eslint-disable-line import/no-import-module-exports
+
+const dmlType = ['INSERT', 'UPDATE', 'DELETE'];
+
+// const exception = () => {
+//   throw new Error(
+//     'Audit log only allows reading and inserting data,'
+//     + ' all modification and removal is not allowed.'
+//   );
+// };
+
+const tryJsonParse = (data) => {
+  const newData = data;
+  if (typeof data === 'object') {
+    Object.entries(data).forEach(([key, value]) => {
```

Comment:
something to be aware of: both ```null``` and an array are ```typeof "object"``` in JS

Reply:
Updated to correctly handle null and array use case.